docs/vocascan-server/traefik/traefik.md

```diff
@@ -0,0 +1,165 @@
+# Installation with Traefik
+
+This guide will get you through the installation of your own vocascan-server paired with Traefik as a reverse proxy and
+Let's Encrypt.
+
+1. First of all you have to install Docker an Docker-Compose. To do this have a look at point 1 and 2 our
+   [guide](vocascan-server/installation#_2-Docker) in the general vocascan-server installation section.
+
+2. After that install `apache2-utils` to create a secure password hash for the Traefik monitoring website with
+   `htpasswd`
+
+   ```bash
+   sudo apt-get install apache2-utils
```

Comment:
if you use docker, you wont like to install some additional packages directly on your system. Instead, you would like to use a docker container to do that job. See how its done in the [deploy a docker registry](https://docs.docker.com/registry/deploying/#native-basic-auth) guide.